scripts/generate_grammar.py

```diff
@@ -249,7 +249,7 @@ def concat_dir(dname, extension, add_line_numbers=False):
 cmd += ["-o", result_source, "-d", target_file]
 print(' '.join(cmd))
 proc = subprocess.Popen(cmd, stderr=subprocess.PIPE)
-res = proc.wait()
+res = proc.wait(timeout=10)
```

Comment:
Be good to add a reason for the timeout i.e. to ensure CI does not hang as was seen when running with Bison 3.x release.

Reply:
done!